ROADMAP.md

```diff
@@ -0,0 +1,43 @@
+# Four Keys 2021 Roadmap
+
+## Mission and Vision
+
+Four Keys mission:
+
+  Be the industry-standard reference implementation for the automated calculation of the DORA Four Keys metrics.  
+
+The vision for this is:
+
+* All discussion of the DORA Four Key metrics center around the key data entities of Changes, Deployments, and Incidents
+* A rich set of tool integrations and data mappings
```

Comment:
And that the rest of the work will be around integrations, different deployment patterns, and ease of use.